src/main/java/com/training/web/pages/flipkart/ResultPage.java

```diff
@@ -1,93 +1,92 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.SelenideElement;
 import com.training.basepages.FlipkartBasePage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.text.NumberFormat;
+import java.text.ParseException;
 import java.util.*;
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
 
 public class ResultPage extends FlipkartBasePage {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    private By lowToHigh = By.xpath("//*[text()='Price -- Low to High']");
-    private By productResults = By.xpath("//img[contains(@class,'_2r_T1I')]");
-    private By shoesPrice = By.className("_30jeq3");
-    private By nextPageButton = By.xpath("//*[text()='Next']");
-    private By products = By.xpath("//div[@class='_312yBx SFzpgZ']");
-
-    public ResultPage (WebDriver driver) {
-        super(driver);
-    }
+    private String lowToHigh = "//*[text()='Price -- Low to High']";
+    private String productResults = "//img[contains(@class,'_2r_T1I')]";
+    private String shoesPrice = "//*[@class='_30jeq3']";
+    private String nextPageButton = "//*[text()='Next']";
+    private String sortBy = "//*[@class='_10Ermr']";
 
     public ResultPage sortLowToHigh() {
-        wait.until(ExpectedConditions.elementToBeClickable(lowToHigh));
-        driver.findElement(lowToHigh).click();
-        wait.until(ExpectedConditions.elementToBeClickable(shoesPrice));
+
+        waitForLoader();
+        $x(lowToHigh).shouldBe(visible).click();
         return this;
     }
 
-    public ArrayList<Integer> getPrice() {
-        waitForLoader();
-        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(shoesPrice));
+    public List<SelenideElement> getPriceString(){
```

Comment:
I am confused by the naming of this method. It says `getPriceString`, but rather it isn't returning a list of strings. it is returning a list of elements.
Probably `getPrice` should be the correct method name